res/css/views/right_panel/_TimelineCard.scss

```diff
@@ -43,62 +43,78 @@ limitations under the License.
     }
 
     .mx_NewRoomIntro {
-        margin-inline-start: var(--BaseCard_EventTile-spacing-horizontal);
-        margin-inline-end: var(--BaseCard_EventTile-spacing-horizontal);
+        margin-inline-start: var(--BaseCard_EventTile-spacing-inline);
+        margin-inline-end: var(--BaseCard_EventTile-spacing-inline);
     }
 
     .mx_EventTile_content {
         margin-right: 0;
     }
 
-    .mx_EventTile:not([data-layout="bubble"]) {
-        &.mx_EventTile_info .mx_EventTile_line,
-        .mx_EventTile_line {
-            padding: var(--BaseCard_EventTile_line-padding-block) var(--BaseCard_EventTile-spacing-horizontal);
-
-            .mx_EventTile_e2eIcon {
-                inset-inline-start: 8px;
+    .mx_EventTile {
+        &[data-layout=bubble] {
+            &::before {
+                z-index: 0; // enable background color on hover
```

Comment:
We should set it to the default value `auto`. This way it kind of „resets“ the `-1` from the other event tile style.